src/Compilers/CSharp/Portable/Syntax/ArgumentSyntax.cs

```diff
@@ -1,17 +1,128 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
+using System.Diagnostics;
+
 namespace Microsoft.CodeAnalysis.CSharp.Syntax
 {
     public partial class ArgumentSyntax
     {
+        /// <summary>
+        /// Get expression representing the value of the argument.
+        /// </summary>
+        public ExpressionSyntax Expression
+        {
+            get
+            {
+                if (RefOrOutKeyword.Kind() != SyntaxKind.OutKeyword)
+                {
+                    return (ExpressionSyntax)ExpressionOrDeclaration;
+                }
+
+                return ExpressionOrDeclaration as ExpressionSyntax;
+            }
+        }
+
+        public ArgumentSyntax WithExpression(ExpressionSyntax expression)
```

Comment:
I couldn't find where this is used. Is this used, or is it there for API consistency?